packages/opentelemetry-types/src/metrics/counter.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface CounterTimeseries {
+  // Adds the given value to the current value. Values cannot be negative.
+  add(value: number): void;
```

Comment:
Maybe a topic for the SIG, but would it be worth putting a comment explaining that `bigint` is not currently supported but may be added in a future release?

Reply:
I think it's a good topic, added to the sig agenda.